modules/cnv-networking-wizard-fields-web.adoc

```diff
@@ -13,10 +13,10 @@
 |Name | Description
 
 |Name
-|Name for the NIC.
+|Name for the Network interface.
```

Comment:
I think this can be lowercase (`network interface`) unless you want to write out `Network Interface Card`